index.html

```diff
@@ -37,9 +37,9 @@ <h1>Highlighted televisions!!!!</h1>
     
     <h1>Televisions</h1>
     <button type="button" id="currentStockBtn" onclick="getAllTelevisions(); ">Show all</button>
```

Comment:
Is er een reden voor dat je hier de ; wel hebt laten staan, maar bij de andere 3 button's deze weggehaald hebt?

Reply:
slordigheid. Tnx! ik moet beter op mijn final result letten